analyses/focal-cn-file-preparation/run-prepare-cn.sh

```diff
@@ -15,21 +17,27 @@ script_directory="$(perl -e 'use File::Basename;
   print dirname(abs_path(@ARGV[0]));' -- "$0")"
 cd "$script_directory" || exit
 
+# Set up exon file
+gunzip -c ../../data/gencode.v27.primary_assembly.annotation.gtf.gz \
+  | awk '$3 ~ /CDS/' \
+  | convert2bed --do-not-sort --input=gtf - \
+  > $exon_file
+
 # Prep the CNVkit data
 Rscript --vanilla -e "rmarkdown::render('00-add-ploidy-cnvkit.Rmd', clean = TRUE)"
 
 # Run annotation step for CNVkit
 Rscript --vanilla 01-prepare-cn-file.R \
   --cnv_file ../../scratch/cnvkit_with_status.tsv \
-  --gtf_file ../../data/gencode.v27.primary_assembly.annotation.gtf.gz \
+  --exon_file $exon_file \
```

Comment:
`exon` -> `cds` whoops. This is my bad. May just want to run through a search and replace.